### ISSUE TITLE
How to berks update [cookbook] without updating its dependencies?

### ISSUE THREAD

**Issue by alexya** [berkshelf/berkshelf#1566]
e.g. 
```
name             'my_nagios'
version          '2.0.7'

depends 'nagios', '~> 7.2.6'
depends 'nrpe', '~> 1.5.2'
recommends 'python', '~> 1.4.6'
```

now, I did a little change to my_nagios cookbook and I only want to update itself without dependencies such as nagios, nrpe, python, etc. as I am sure that I don't need upgrade dependency cookbook's version.

the following command will return "Resolving cookbook dependencies...", and there is no changes to berksfiel.lock. If I remove the parameters "-o my_nagios", it will update berksfile.lock with all its dependencies. 
```
berks update my_nagios -o my_nagios 
```

In summary, how to berks update [cookbook] without updating its dependencies? 
Any help will be much appreciated.

p.s. I used Google, but can't find the answer.

**Comment by lamont-granquist** [comment]
This is a case for editing the lockfile directly and only bumping the one version constraint.  I don't think bundler provides any API to do this either outside of directly editing Gemfile.lock which is what I always do.

**Comment by ehaselwanter** [comment]
@lamont-granquist came across your answer. still true I guess? should we add that as a sort of 'edge case/danger section' in the readme?
